save-to-webm/main.go

```diff
@@ -66,8 +68,16 @@ func (s *webmSaver) PushOpus(rtpPacket *rtp.Packet) {
 			return
 		}
 		if s.audioWriter != nil {
-			s.audioTimestamp += sample.Duration
-			if _, err := s.audioWriter.Write(true, int64(s.audioTimestamp/time.Millisecond), sample.Data); err != nil {
+			if !s.hasStartAudioOffset {
+				s.startAudioOffset = sample.PacketTimestamp
```

Comment:
Actually I think for the very first packet received, I should be using the old value (i.e. `sample.Duration/time.Millisecond), instead of forcing it to 0 (as we're setting `startAudioOffset` to `sample.PacketTimestamp` and then immediately subtracting the former from the latter below.

In other words, the algo would be:
- 1st packet: `int64(sample.Duration/time.Milisecond)`
- nth packet: `int64(sample.PacketTimestamp) - int64(s.startAudioOffset)`